src/utils/aeternity.js

```diff
@@ -68,10 +69,12 @@ export const scanForWallets = async () => {
     connectionInfo: { id: 'spy' },
   });
   const detector = await Detector({ connection: scannerConnection });
+  const webWalletTimeout = setTimeout(() => store.commit('useIframeWallet'), 2000);
```

Comment:
Timeout is a tricky part, it depends on extension load time. In my case, 2 seconds is enough. I don't think that we can check it in some other way.

Reply:
Lets see how it behaves when testing. Maybe we can think of other way for detection with intervals and fallback after some time 🤔

Reply:
We can hide iframe if wallet appear later than 2 sec 🙃

Reply:
In terms of UX I'd rather prefer to have some sort of notification (popup) presented to the user so they can confirm wether they want to switch to using the extension. But if we get to agreement for specification and implementation of #606 - then this problem should be obsolete as if you have extension installed should be able to sync it with the integrated wallet. 💭